scripts/dist.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+mvn clean package
+
+TO_DIR=dist/
+rm -rf $TO_DIR
+mkdir -p $TO_DIR
+cp scripts/conduit-connector-s3-iceberg $TO_DIR
+cp target/conduit-connector-s3-iceberg-*.jar $TO_DIR/libs
```

Comment:
maybe we should create the `/libs` directory first? I tried running the script and "libs" was a runnable file, not a folder, maybe I did sth wrong not sure

Reply:
done